docs/en/02_Features/pdf_export.md

```diff
@@ -128,23 +117,6 @@ folder.
 
 To customise the footer of the PDF, you can modify the `Page_pdffooter.ss` template in your theme.
 
-## Defining the version of wkhtmltopdf to use
-
-We currently provide two versions of wkhtmltopdf for CWP customers to use.
-
-The version that is used by default is version 0.9.6, and is an older, stable version that doesn't support as many
-features, but has been tested to work well with the default CWP templates.
-
-Some CWP customers have requested an updated version, and so we have also installed version 0.12.1.1. In order to use
-this updated version, you must define the binary to use.
-
-Since CWP Recipe release 1.0.4, the only configuration that is needed is an addition to your YAML configuration:
-
-```yaml
-CWP\CWP\PageTypes\BasePage:
-  wkhtmltopdf_binary: '/usr/local/bin/wkhtmltopdf_12'
```

Comment:
Should we deprecate the configuration flag for this as well?

Reply:
Yep, we should switch to `/usr/local/bin/wkhtmltopdf`. I'll have a look at changing that.